java/src/jmri/jmrit/withrottle/DeviceManager.java

```diff
@@ -9,14 +8,43 @@
  *
  * @author Randall Wood Copyright 2011, 2017
  */
-public interface DeviceManager extends RosterGroupSelector {
+public interface DeviceManager {
```

Comment:
Is there a reason this no longer extends RosterGroupSelector but contains the methods that the RosterGroupSelector interface contains?

Reply:
1) The RosterGroupSelector interface required the implementation of several property change listeners that were all dummy values
2) the RosterGroupSelector interface requires implementation of a property change listener, but there really isn’t a property change listener here.
3) the method name for retrieving the roster group is used elsewhere, so I wanted to keep that.

Note that this also allows the headless server to send roster groups, which was not previously possible.